lib/src/helpers/enums.dart

```diff
@@ -227,3 +227,71 @@ abstract final class Device {
   static final String propertyPrefix =
       isFirefox ? 'moz' : (isIE ? 'ms' : (isOpera ? 'o' : 'webkit'));
 }
+
+/// HTTP status codes.
```

Comment:
nit: I'd add a non-dartdoc comment here briefly mentioning that this is a copy of the `dart:_internal` version that's only exposed through `dart:html` and `dart:io`.

Reply:
Updated.